src/libraries/icubmod/usbCamera/linux/V4L_camera.cpp

```diff
@@ -160,6 +226,47 @@ bool V4L_camera::open(yarp::os::Searchable& config)
     return true;
 }
 
+
+int V4L_camera::getRgbHeight(){
+    return height();
+}
+
+int V4L_camera::getRgbWidth(){
+    return width();
+}
+
+bool V4L_camera::setRgbResolution(int width, int height){
+    param.width=width;
+    param.height=height;
+    return true;
+}
+
+bool V4L_camera::getRgbFOV(double &horizontalFov, double &verticalFov){
+    horizontalFov=param.horizontalFov;
+    verticalFov=param.verticalFov;
+    return configFx && configFy;
+}
+
+bool V4L_camera::setRgbFOV(double horizontalFov, double verticalFov){
+    param.horizontalFov=horizontalFov;
+    param.verticalFov=verticalFov;
+    return true;
```

Comment:
Ops, I forgot to say this method shall actually change the resolution of the video streaming, so it should call the `deviceUninit` function to stop the streaming, then change the param setting and then restart the streaming video with 'deviceInit'.